test/school_house_web/views/html_helpers_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule SchoolHouseWeb.HtmlHelpersTest do
+  use ExUnit.Case, async: true
+
+  alias SchoolHouseWeb.HtmlHelpers
+
+  describe "avatar_url/1" do
+    test "appends .png to GitHub profile link" do
+      assert "https://github.com/elixirschool.png" == HtmlHelpers.avatar_url("https://github.com/elixirschool")
+    end
+  end
+
+  describe "current_page_locale_path/2" do
+    test "returns the current page path for another locale" do
+      Gettext.put_locale("es")
+
+      assert "fr/ecto/changesets" ==
+               HtmlHelpers.current_page_locale_path(%Plug.Conn{request_path: "es/ecto/changesets"}, "fr")
```

Comment:
This test doesn't reflect real world usage, as `request_path` will include the leading slash: `/es/ecto/changesets`.

So the test should be:

```elixir
assert "/fr/ecto/changesets" ==
               HtmlHelpers.current_page_locale_path(%Plug.Conn{request_path: "/es/ecto/changesets"}, "fr")
```

Reply:
I updated the tests to use the ConnTest and `build_conn` to make this more accurate and less reliant on building things up manually which opens up opportunities for issues like this.